### ISSUE TITLE
Migration gdpr SQL Server error with FALSE value 

### ISSUE THREAD

**Issue by tsdogs** [2amigos/yii2-usuario#314]
I encounter problems after #211 has been solved.
Then I see another issue which seems to be the opposite: #313 

This commit is causing problems with SQL Server, error is:

The name "FALSE" is not permitted in this context. Valid expressions are constants, constant expressions, and (in some contexts) variables. Column names are not permitted.

ALTER TABLE [user] ADD [gdpr_consent] bit DEFAULT FALSE

Either use 'FALSE' or 0.

ALTER TABLE [user] ADD [gdpr_consent] bit DEFAULT 'FALSE'

ALTER TABLE [user] ADD [gdpr_consent] bit DEFAULT 0

are both correct for SQL Server.

Maybe this cloud probably be raised as bug to yii2.
In the mean time maybe having a migration class which solves this could be an option.

**Comment by maxxer** [comment]
So TSQL wants FALSE as *string* or a numerical zero?

**Comment by tsdogs** [comment]
That seems so for sql server:
https://stackoverflow.com/questions/40244152/how-to-add-a-boolean-datatype-column-to-an-existing-table-in-sql

**Comment by tonydspaniard** [comment]
@tsdogs do you have time to correct this issue for both postgress and SQL server? I suggest this as you probably have the environment configured on your computer. That would be most helpful.

**Comment by tsdogs** [comment]
Ok, though I also use freetds for sql server access so this might be corrected somewhere else.
I'll try to find a solution in the week end.

**Comment by tsdogs** [comment]
I'm trying to solve this. I have implemented a method in MigrationHelper::getBooleanValue() which should be used when defaultValue is applied to a boolean field, though I feel it would better to implement  a Migration class which would override the defaultValue() function (like it was in dektrium implementation)...
